.github/workflows/app.yml

```diff
@@ -21,3 +21,17 @@ jobs:
         run: |
           flutter pub get
           flutter pub publish --dry-run
+  
+  build_js:
+    runs-on: ubuntu-latest
+    container: docker.io/emscripten/emsdk:latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
nitpick but can you use @v4 for actions/* you have added today?

Reply:
helps with some of our warnings

Reply:
We have some other v3 on old files. Is there any reason to not upgrade them?